src/HeatCondEquationSystem.C

```diff
@@ -729,7 +729,7 @@ HeatCondEquationSystem::register_wall_bc(
     stk::mesh::put_field_on_mesh(*alphaField, *part, nullptr);
   
     // aux algs
-    AuxFunctionAlgorithm *alphaAuxAlg;
+    AuxFunctionAlgorithm *alphaAuxAlg = NULL;
```

Comment:
I suppose this doesn't have anything to do with `mesh_motion`?

Reply:
no. this is a compiler warning fix from a previous commit. I actually forgot to rebase resulting in a merge commit which in turn resulted in a previous commit showing up in this pull request, although the actual file changes only correspond to the last commit in the pull request.